sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -155,14 +155,15 @@ class QueryExecution(
     append("== Parsed Logical Plan ==\n")
     QueryPlan.append(logical, append, verbose, addSuffix, maxFields)
     append("\n== Analyzed Logical Plan ==\n")
-    val analyzedOutput = try {
-      truncatedString(
-        analyzed.output.map(o => s"${o.name}: ${o.dataType.simpleString}"), ", ", maxFields)
+    try {
```

Comment:
Cam you explain why it is printing the exception twice?

Reply:
First time it is printed when AnalysisException come while trying to find output schema
https://github.com/apache/spark/blob/4021354b73dd86ee765f50ff90ab777edfc21bdb/sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala#L137-L142
Second time while generating a tree string for analyzed plan
https://github.com/apache/spark/blob/4021354b73dd86ee765f50ff90ab777edfc21bdb/sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala#L145